### PR TITLE
docs(examples): add react-spring example

### PR DESCRIPTION
**Pull request by Girish21**
A simple example for integrating `react-spring` with Remix

### CODE REVIEW

**Review by LANCENET — APPROVED** [review]
😎

**Comment by almamariali** [comment]
> 😎

File: examples/react-spring/app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <Links />
+        <noscript>
+          <link rel="stylesheet" href={noScriptStyles} />
+        </noscript>
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        {process.env.NODE_ENV === "development" && <LiveReload />}
```
Comment (MichaelDeBoey) [inline examples/react-spring/app/root.tsx:36]:
```suggestion
        <LiveReload />
```
See #1352

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by kentcdodds — APPROVED** [review]
Looks good to me! Thanks 👏👏

**Pull request merged**